pkg/process/procutil/data_scrubber_fallback.go

```diff
@@ -18,3 +21,12 @@ var (
 
 	forbiddenSymbolsRegex = "[^a-zA-Z0-9_*]"
 )
+
+func (ds *DataScrubber) stripArguments(cmdline []string) []string {
+	// We will sometimes see the entire command line come in via the first element -- splitting guarantees removal
+	// of arguments in these cases.
+	if len(cmdline) > 0 {
+		return []string{strings.Split(cmdline[0], " ")[0]}
```

Comment:
You can use `splitN` here as it's only concerned with the command that is running.

Reply:
Thanks! Missed this one - will fix in next commit!